installing/install_config/enable-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,16 @@
+[id="enable-cluster-wide-proxy"]
+= Configuring cluster-wide proxy
+include::modules/common-attributes.adoc[]
+:context: enable-cluster-wide-proxy
+
+toc::[]
+
+Production environments can deny direct access to the Internet and instead have
```

Comment:
@danehans I pulled this paragraph from the 3.11 docs, but wasn't sure if we needed to change the "as simple as setting standard environment variables in configuration or JSON files" part for 4.2. Suggestions?